pkg/terraformer/config.go

```diff
@@ -101,6 +101,18 @@ func (t *terraformer) InitializeWith(initializer Initializer) Terraformer {
 	return t
 }
 
+//GetState returns the contenten of terraform state config map
```

Comment:
```suggestion
// GetStateConfigMapAsRawState returns the content of terraform state config map.
```

I think `GetStateConfigMapAsRawState ` is a better self-explaining name.

Reply:
At the moment we are using config map but if we change it to secret we have to change the name of the method. No matter what kind is the k8s resource in which we store the state we get the same RawState object.